cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -680,6 +681,9 @@ func checkEndpointReachable(
 		return false, fmt.Errorf("unable to prepare reachability request: %v, error: %s", request, err)
 	}
 
+	if logrus.GetLevel() >= logrus.DebugLevel {
```

Comment:
What do you think of using the logger entity that is used by the HttpTracer to check the level?

```suggestion
	if wlog.IsLevelEnabled(logrus.DebugLevel) {
```

Reply:
much better, thanks!. Changing it